bazel/scripts/run_integ_tests.sh

```diff
@@ -299,10 +299,11 @@ run_test() {
 }
 
 create_xml_report() {
-    rm -f "${MERGED_REPORT_FOLDER}/"*.xml
-    mkdir -p "${MERGED_REPORT_FOLDER}"
-    python3 lte/gateway/python/scripts/runtime_report.py -i "[^\/]+\.xml" -w "${INTEGTEST_REPORT_FOLDER}" -o "${MERGED_REPORT_FOLDER}/integtests_report.xml" 
-    rm -f "${INTEGTEST_REPORT_FOLDER}/"*.xml
+    local MERGED_REPORT_XML="integtests_report.xml"
+    rm -f "${MERGED_REPORT_FOLDER}/${MERGED_REPORT_XML}"
+    mkdir -p "${INTEGTEST_REPORT_FOLDER}"
+    python3 lte/gateway/python/scripts/runtime_report.py -i "[^\/]+\.xml" -w "${INTEGTEST_REPORT_FOLDER}" -o "${MERGED_REPORT_FOLDER}/${MERGED_REPORT_XML}" 
+    sudo rm -f "${INTEGTEST_REPORT_FOLDER}/"*.xml
```

Comment:
just curious: why was the `sudo` not needed before?

Reply:
As far as I understand:
`/var/tmp/test_results/` is created with permission 777. The XML files were there before and (forced) deletion was allowed. Now the `INTEGTEST_REPORT_FOLDER` changed to `/var/tmp/test_results/integtest_reports` and this directory has normal permissions. The XML file is created by sudo, because the tests are executed as sudo, therefore you will get a `Permission denied` error if you try to remove them.